README.md

```diff
@@ -117,10 +117,40 @@ use YieldStudio\NovaGoogleAutocomplete\AddressMetadata;
 use YieldStudio\NovaGoogleAutocomplete\GoogleAutocomplete;
 
 // This autocomplete field will return results that match a business name instead of address.
-// All the same address data is still stored.  
+// All the same address data is still stored.
 GoogleAutocomplete::make('Address')->placeType('establishment');
 ```
 
+### Capturing JSON response
+
+If you want to capture the entire response as a JSON object, you can use the `fromResponse()` helper instead of using `fromValues()`.
+This helper will capture the entire response containing all the requested values as a JSON string.
```

Comment:
```suggestion
### Capturing all values as JSON

If you want to capture all requested values as a JSON object, you can use the `fromValuesAsJson()` helper instead of using `fromValue()`.
```